singer-io/tap-bing-ads
# TDL-9902 removed the automatic fields from ad ext report

# Description of change
Removed the constraint imposed on AdExtensionDetailReport of having 'Ctrl', 'Clicks', and 'Impressions' from the required fields in the reports.py

# Manual QA steps
 - Check that the automatic fields test runs without any failures
 - Check that the catalog generated has available inclusion property for Ctr, Clicks, and Impression fields for AdExtensionDetailReport 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
